ci/vpp_test_setup/vpp_host.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+kubectl delete ns vpp
+kubectl create ns vpp
+kubectl create -n vpp configmap vpp-startup-config --from-file=/runner/_work/userspace-cni-network-plugin/userspace-cni-network-plugin/docker/vpp-centos-userspace-cni/startup.conf
+
+worker="kind-control-plane"
+
+
+rm /opt/cni/bin/userspace
+docker exec -i kind-control-plane bash -c "mkdir -p /var/run/vpp/app"
+
+cat << EOF | kubectl apply -f -
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: userspacecni-$worker
+  namespace: vpp
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: $worker
+  containers:
+  - name: userspacecni-$worker
+    image: userspacecni:latest
+    imagePullPolicy: IfNotPresent
+    volumeMounts:
+    - name: cni
+      mountPath: /opt/cni/bin
+  volumes:
+    - name: cni
+      hostPath:
+        path: /opt/cni/bin
+  restartPolicy: Never
+EOF
+
+
+echo "sleeping for 20 to allow userspace to deploy first"
+sleep 20
```

Comment:
same as previous, use the ``ready``, in case userspace does take over 20 seconds for once?

Reply:
see above